portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -26,7 +26,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = request.getParameter("email-input");
     String textValue = request.getParameter("text-input");
     long timestamp = System.currentTimeMillis();
-
+    Document doc =
+    Document.newBuilder().setContent("textValue").setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
```

Comment:
nice! looks like you got NLP to work with datastore?

Reply:
Yes, got help from PA in technical chat.  It was a complex bug in dependencies management.